### PR TITLE
fix cli tool

### LINKED ISSUE #6386: Can't start Hyper from terminal (/opt/homebrew/bin/hyper: line 4: /usr/bin/python: No such file or directory)
- [x] I am on the [latest](https://github.com/vercel/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/vercel/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**:  MacOS Monterey 12.3
- **Hyper.app version**: 3.2.0
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: [.hyper.js](https://gist.github.com/GBrachetta/0bb163e8429f9e72f73ea0faf68a4152)
- **Relevant information from devtools** _(CMD+ALT+I on macOS, CTRL+SHIFT+I elsewhere)_: 

## Issue

Attempting to invoke Hyper from the terminal (terminal.app or iTerm2) fails:

```bash
$ hyper

$ /opt/homebrew/bin/hyper: line 4: /usr/bin/python: No such file or directory
$ /opt/homebrew/bin/hyper: line 8: ./MacOS/Hyper: No such file or directory
```

Please note my Python installation was done via Homebrew and as I am on an M1 machine all Homebrew installs go under `/opt/homebrew/bin`.

Also, there's no such `./MacOS/Hyper` anywhere. Was it supposed to get created by Hyper? It's not there.

Hyper was also installed with Homebrew.

Maybe Related: 

I cannot run either `Tools/Install Hyper CLI command in PATH`, I get the following error:

Screenshot 2022-03-15 at 14 24 35

### PR DESCRIPTION
**Pull request by vtj-ttanaka**
fix #6386

### CODE REVIEW

**Comment by zcutlip** [comment]
Unfortunately this isn't a viable solution since python3 isn't installed by default on macOS.

See https://github.com/vercel/hyper/issues/6386#issuecomment-1071037419

File: build/mac/hyper [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 # Deeply inspired by https://github.com/Microsoft/vscode/blob/1.17.0/resources/darwin/bin/code.sh
 
-function realpath() { /usr/bin/python -c "import os,sys; print(os.path.realpath(sys.argv[1]))" "$0"; }
+function realpath() { /usr/bin/python3 -c "import os,sys; print(os.path.realpath(sys.argv[1]))" "$0"; }
```
Comment (zcutlip) [inline build/mac/hyper:4]: How about
```sh
# dash compatible function, anticipating bash deprecation
_realpath() { cd "$(dirname $0)" && echo "$(pwd)/$(basename $0)"; }

```

Comment (vtj-ttanaka) [reply build/mac/hyper:4]: cool. this change is fine.

**Comment by vtj-ttanaka** [comment]
fix for #6389 is correct. so i whill this PR closed.

**Pull request closed without merging**